### PR TITLE
Android guideline improvements part I

### PR DESCRIPTION
**Pull request by fmontesino**
Here's some Android guideline improvements for the first section (1. Easy to perceive) of the guidelines. More to come!

### CODE REVIEW

**Review by geftimov — APPROVED** [review]

**Review by KaneCheshire — APPROVED** [review]
This is great 👏

**Review by voicu-ynap — APPROVED** [review]

**Review by kleinsenberg — APPROVED** [review]

File: guidelines/1.4.10.md [hunk 1]
```diff
@@ -105,7 +105,15 @@ override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollect
 
 ## Guidance for Android
 
-This section needs more content. [Contribute via Github](https://github.com/theappbusiness/accessibility-guidelines/) or [email](mailto:jeanfrancois@theappbusiness.com).
+All Views will automatically allocate its content, as long as there's no `height` restriction. Using `android:layout_height="wrap_content"` is strongly recommended whenever possible.
+
+If you're using a `ConstraintLayout`, make sure it has `match_parent` on both **height** and **width** to fill the whole viewport. Make sure all its child Views are properly constraint to the edges.
```
Comment (kleinsenberg) [inline guidelines/1.4.10.md]: Minor: `constrained` instead of `constraint`? 🤔

Comment (fmontesino) [reply guidelines/1.4.10.md]: Changed!

**Comment by fmontesino** [comment]
I think this PR might fix this? https://github.com/theappbusiness/accessibility-guidelines/issues/63

**Pull request merged**